packages/server/src/api/controllers/auth.ts

```diff
@@ -45,9 +45,9 @@ export async function fetchSelf(ctx: UserCtx) {
     try {
       const userTable = await sdk.tables.getTable(InternalTables.USER_METADATA)
       // specifically needs to make sure is enriched
-      ctx.body = await outputProcessing(userTable, user as Row)
+      ctx.body = (await outputProcessing(userTable, user as Row)) as ContextUser
```

Comment:
If we remove the `as Row`, is the `as ContextUser` required?

Reply:
Silly - should have checked, didn't realise `outputProcessing` can handle context user directly!